creator-node/src/routes/audiusUsers.js

```diff
@@ -14,7 +14,15 @@ module.exports = function (app) {
     const metadataJSON = req.body.metadata
 
     const metadataBuffer = Buffer.from(JSON.stringify(metadataJSON))
-    const { multihash, fileUUID } = await saveFileFromBuffer(req, metadataBuffer, 'metadata')
+    let multihash, fileUUID
```

Comment:
is this just to make the error message more specific?

Reply:
yes!